schemas/common/descriptors/search/searchdescriptor.schema.json

```diff
@@ -0,0 +1,92 @@
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/descriptors/searchdescriptor",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Descriptor",
+  "meta:extensible": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/common/desciptors/schemadescriptor#/definitions/descriptor"
+  ],
+  "meta:abstract": false,
+  "type": "object",
+  "description": "A descriptor that describes search annotations for specific properties of the schemas.",
+  "definitions": {
+    "searchdescriptor": {
+      "properties": {
+        "search:indexable": {
+          "title": "isIndexable",
+          "type": "boolean",
+          "description": " When present, Boolean indicating if the property is indexable / searchable"
+        },
+        "search:fetchable": {
+          "title": "is Fetchable",
+          "type": "boolean",
+          "description": "When present, Boolean indicating if the property is returned by default in search query results"
+        },
+        "search:filterable": {
+          "title": "is Filterbale",
+          "type": "boolean",
+          "description": "When present, Boolean indicates if this property can be used as  a filter in search queries"
+        },
+        "search:stored": {
+          "title": "is stored",
+          "type": "boolean",
+          "description": "When present, Boolean indicating if the property's raw values are stored in the index"
+        },
+        "search:sort": {
+          "properties": {
+            "search:sortable": {
+              "title": "isSortable",
+              "type": "boolean",
+              "description": " When present, Boolean indicating if the property is sortable "
+            },
+            "search:caseinsensitive": {
+              "title": "is Case Insensitive",
+              "type": "boolean",
+              "description": "When present, Boolean indicating if the sort on the property is case insensitive"
+            }
+          }
+        },
+        "search:analyzer": {
+          "properties": {
+            "search:analyzed": {
+              "title": "isAnalyzed",
+              "type": "boolean",
+              "description": " When present, Boolean indicating if the property is analyzed "
+            },
+            "search:analyzer": {
+              "title": "Analyzer to be applied",
+              "type": "array",
+              "items": {
+                "type": "string",
+                "meta:enum": {
+                  "standard": "standard",
+                  "keyword": "keyword",
+                  "simple": "simple"
+                },
+                "description": "When present, value of enum literals in array indicates the type of analyzers to be applied"
```

Comment:
is this a closed list?

Reply:
Yes, this is a closed list - we may enhance it later to add others. If clients need special analyzers, they will have to work with us to enable those.